lib/logger.ts

```diff
@@ -28,6 +28,8 @@ import {Writable} from 'stream';
 import {LEVEL, MESSAGE} from 'triple-beam';
 import winston from 'winston';
 import LokiTransport from 'winston-loki';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
 import {Papertrail} from 'winston-papertrail';
```

Comment:
I did the full
```
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
```
because that's what I saw around. Only later did I learn that it's probably better to decorate with `@ts-expect-warning|error`, as it would complain if something changed and the next line would no longer emit a ts problem.   Maybe we'll do a wide grep+replace someday.